.changeset/tender-emus-eat.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': minor
```

Comment:
```suggestion
'astro': patch
```

This should be a patch as it fixes a missing flag description.

Reply:
Thank you!